index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <h1>Hello World, we are software developers!</h1>
+    <h1 id="title">Title</h1>
```

Comment:
avoid having two `h1` tags in the same element. you can convert the second `h1` tag into a `h2` or `p` tag

Reply:
> * [ ]  avoid having two `h1` tags in the same element. you can convert the second `h1` tag into a `h2` or `p` tag
- [ ] Kindly phrase the comments politely, as `suggestions` instead of `orders` for example 
* This is a polite suggestion: *"I think it would be better to use semantic tags"*
* This reads like an order: *"Use semantic tags"*

- [ ] When giving a review, always include an explanation, a "why", for each requested change. For example
* This is a well-explained request: *Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible*
 * This request lacks a "why": *Why don't you use an article tag here instead of a div?*